packages/core/cli/src/scripts/createProject/index.ts

```diff
@@ -1,22 +1,43 @@
 import path from 'path';
+const shell = require('shelljs');
 import log from '../../utils/log';
-import copyIntegrationTheme from './copyIntegrationTheme';
-import copyAgnosticTheme from './copyAgnosticTheme';
 import processMagicComments from './processMagicComments';
+import * as process from 'process';
 
-async function createProject(integration: string, targetPath: string): Promise<void> {
-  log.info(`Coppying agnostic theme to ${targetPath}`);
-  await copyAgnosticTheme(integration, targetPath);
+interface ICreateProjectProps {
+  integration: string,
+  targetPath: string,
+  repositoryLink: string
+}
 
-  log.info(`Coppying ${integration}-theme to ${targetPath}`);
-  await copyIntegrationTheme(integration, targetPath, ['_theme', '.nuxt', 'node_modules']);
+async function createProject({
+  integration,
+  targetPath,
+  repositoryLink
+}: ICreateProjectProps): Promise<void> {
+  const templatePath = `${targetPath}/${integration}`;
+  try {
+    shell.rm('-rf', templatePath);
```

Comment:
I don't think it work on Windows

Reply:
Hey, according to doc of [shelljs](https://github.com/shelljs/shelljs#shelljs---unix-shell-commands-for-nodejs) - "ShellJS is a portable (Windows/Linux/macOS) implementation of Unix shell commands on top of the Node.js AP", so it should work.

Reply:
rimraf